### PR TITLE
Correct clipboard read/write version numbers.

### PR DESCRIPTION
**Pull request by jpmedley**
https://www.chromestatus.com/feature/5074658793619456
https://www.chromestatus.com/feature/5861289330999296

### CODE REVIEW

File: api/Clipboard.json [hunk 1]
```diff
@@ -37,9 +51,16 @@
...
+          "webview_android": [
+            {
+              "version_added": "84"
+            },
+            {
+              "version_added": "66",
+              "partial_implementation": true,
+              "notes": "Images are not supported on read() and write()."
+            }
+          ],
```
Comment (Elchi3) [inline api/Clipboard.json:63]:
```suggestion
          ]
```

File: api/Clipboard.json [hunk 2]
```diff
@@ -100,9 +135,16 @@
...
+            "webview_android": [
+              {
+                "version_added": "84"
+              },
+              {
+                "version_added": "66",
+                "partial_implementation": true,
+                "notes": "Images are not supported."
+              }
+            ],
```
Comment (Elchi3) [inline api/Clipboard.json:147]:
```suggestion
            ]
```

File: api/Clipboard.json [hunk 3]
```diff
@@ -214,9 +270,16 @@
...
```
Comment (Elchi3) [inline api/Clipboard.json:282]:
```suggestion
            ]
```

**Comment by jpmedley** [comment]
Please take another look.

**Pull request merged**